lib/path/tests/normalise.nix

```diff
@@ -0,0 +1,34 @@
+# Used by ./prop.sh
+#
+# stringsDir is a flat directory containing files with randomly-generated
+# path-like values
+#
+# This file should return a { <value> = <lib.path.subpath.normalise value>; }
+# attribute set for each value. If `normalise` fails to evaluate,
+# "" is returned instead. If not, the result is normalised again and returned
+# too
```

Comment:
This comment really confused me. This suggestion is based on the proposal to just pass the list of strings directly.
```suggestion
# Given a list of path-like strings, return a list of attribute sets of the following form:
#
#     { <string> = <lib.path.subpath.normalise string>; }
#
# If `normalise` fails to evaluate, the attribute value is set to `""`.
# If not, the resulting value is normalised again and an appropriate attribute set added to the output list.
```